sw/host/opentitanlib/src/io/emu.rs

```diff
@@ -35,6 +37,35 @@ pub enum EmuValue {
     FilePathList(Vec<String>),
 }
 
+impl From<&String> for EmuValue {
```

Comment:
Shouldn't this be `String`? Callers can write `.clone().into()`, avoiding unnecessary clones if the caller could simply give up ownership of the string instead.